services/shuttle-crontab/README.md

```diff
@@ -0,0 +1,36 @@
+# shuttle-crontab
+
+A service that calls URLs at specified cron intervals.
+
+# Usage
+
+Create a new `CrontabService` by providing a `shuttle_persist::PersistInstance`
+and an `axum::Router`:
+
+```
+// main.rs
+
+#[shuttle_runtime::main]
+async fn crontab(#[Persist] persist: PersistInstance) -> ShuttleCrontab {
+    let router = Router::new().route("/", get(my_own_route));
```

Comment:
nit: `my_own_route` is great, but a closure is better `|| async { println!("triggered by the crontab service"); }` makes more sense. Maybe this example can be transferred as a crate public doc.

Reply:
The closure makes sense, although I used a different string to make it obvious that this is a route that is going to be mounted next to the cron service's.